script/bulk_import/generic_bulk.rb

```diff
@@ -1,65 +1,201 @@
 # frozen_string_literal: true
 
-require_relative "base"
-require "sqlite3"
-require "json"
+begin
+  require_relative "base"
+  require "sqlite3"
+  require "json"
+rescue LoadError
+  STDERR.puts "",
+              "ERROR: Failed to load required gems.",
+              "",
+              "You need to enable the `generic_import` group in your Gemfile.",
+              "Execute the following command to do so:",
+              "",
+              "\tbundle config set --local with generic_import && bundle install",
+              ""
+  exit 1
+end
 
 class BulkImport::Generic < BulkImport::Base
   AVATAR_DIRECTORY = ENV["AVATAR_DIRECTORY"]
   UPLOAD_DIRECTORY = ENV["UPLOAD_DIRECTORY"]
 
-  def initialize(db_path)
+  def initialize(db_path, uploads_db_path = nil)
     super()
-    @db = create_connection(db_path)
+    @source_db = create_connection(db_path)
+    @uploads_db = create_connection(uploads_db_path) if uploads_db_path
   end
 
   def start
     run # will call execute, and then "complete" the migration
 
     # Now that the migration is complete, do some more work:
 
-    Discourse::Application.load_tasks
-
-    puts "running 'import:ensure_consistency' rake task."
-    Rake::Task["import:ensure_consistency"].invoke
+    # Discourse::Application.load_tasks
+    #
+    # puts "running 'import:ensure_consistency' rake task."
+    # Rake::Task["import:ensure_consistency"].invoke
```

Comment:
I see this has been moved to `BulkImport::Base` as instructions to be run  manually. Why?

Reply:
Oh, let me re-enable that again. I had disabled it because there were problems with long running tasks in the rake task, but it that's mostly solved. One less thing to worry about.